### DISCUSSION TITLE
1.3.5 compile errors in Visual Studio Code

### DISCUSSION THREAD

**Discussion by arnostolk** [Jyers/Marlin#1290] [Q&A] [answered]
Hi,
First of all a very big thank you for providing this firmware repo. I'm really enjoying it and have been updating to latest versions regularly and without any problems.
However, since the latest version, 1.3.5, Visual Studio Code with the Auto Build Marlin plugin generates errors when building.
It throws a number of errors but the initial one states:
`Compiling .pio\build\STM32F103RET6_creality\lib874\Wire\utility\twi.c.o
In file included from .pio\libdeps\STM32F103RET6_creality\SoftwareSerialM\HAL_softserial_STM32.cpp:27:
.pio\libdeps\STM32F103RET6_creality\SoftwareSerialM\HAL_softserial_STM32.h:37:36: error: variable or field 'SoftSerial_Handler' declared void
   37 | extern "C" void SoftSerial_Handler(stimer_t *htim);
      |                                    ^~~~~~~~
.pio\libdeps\STM32F103RET6_creality\SoftwareSerialM\HAL_softserial_STM32.h:37:36: error: 'stimer_t' was not declared in this scope; did you mean 'timer_t'?
   37 | extern "C" void SoftSerial_Handler(stimer_t *htim);
      |                                    ^~~~~~~~
      |                                    timer_t`

I'm not sure what's happening but checking out the older 1.3.4 version from git solves the issue so it looks like it's specific to the latest version. Does anybody has an idea on how to fix this?

**Comment by darco415** [comment, marked as answer]
I normally dont use AutoBuild but I tried it just now and it compiled fine. My previous build using VS along with the config template built fine as well. You could always start fresh and re-clone.

**Reply by arnostolk** [reply]
Thanks for the reply. I will try with a clean git pull. Glad to know it's probably a PEBCAK :-)

**Comment by arnostolk** [comment]
Deleting the .pio folder fixed it. Thanks again!
Arno
